python/pyiceberg/cli/console.py

```diff
@@ -65,14 +65,7 @@ def run(ctx: Context, catalog: str, verbose: bool, output: str, uri: Optional[st
         ctx.obj["output"] = JsonOutput(verbose=verbose)
 
     try:
-        try:
-            ctx.obj["catalog"] = load_catalog(catalog, **properties)
-        except ValueError as exc:
-            if not uri:
```

Comment:
Mostly because it is at a very high level, and the `ValueError` is very broad. For example, if you would provide a credential without a semicolon, it would throw a ValueError:
```
>>> client, secret = "abc".split(":")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: not enough values to unpack (expected 2, got 1)
```
Next to that we currently only check if the `uri` is set from the CLI (and ignore if it is passed in from the config), it will just complain that the URI is not set (while your credential is not having a `:`).